cli/viam/main.go

```diff
@@ -765,6 +758,10 @@ func main() {
 							{
 								Name:  "shell",
 								Usage: "start a shell on a robot part",
+								// TODO: remove this warning
+								Description: `Functionality of the shell command is highly experimental. In particular, there may be text-input issues
```

Comment:
In the description, you might want to mention that you require a `Service` of type `shell` for this to work. Otherwise the error message is not obvious

Reply:
Good call out; added.